### PR TITLE
Show excerpt on blog post list

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

Changes proposed in this pull request:
- Added excerpt to blog posts.
- Adjusted `milv.config.yaml` and changed blue border of post to lightGray.
- Added info about excerpt in docs and fixed relative links in blog posts.

**Related issue(s)**
See also #167

### CODE REVIEW

File: src/components/blog/Post.js [hunk 1]
```diff
@@ -11,15 +12,45 @@ const PostWrapper = styled.div`
   border: 0;
   padding: 0;
   padding: 40px 0 40px;
-  border-bottom: 3px solid ${colors.blue};
+  ${props =>
+    props.borderBottom ? `border-bottom: 3px solid ${colors.lightGray}` : null}
```
Comment (pkosiec) [inline src/components/blog/Post.js]: It's not the same:
screen shot 2018-12-27 at 12 34 41

File: docs/write-blog-posts.md [hunk 2]
```diff
@@ -33,6 +33,8 @@ Replace these parameters with real values:
 - `{AUTHOR}` is your name with an optional position name.
 - `{CONTENT}` is written in Markdown and/or HTML.
```
Comment (klaudiagrz) [inline docs/write-blog-posts.md]:
```suggestion
- `{CONTENT}` is written in Markdown and/or HTML. The content must include the `<!-- overview -->` comment which indicates the part of the blog post that will be displayed on the main page. The **Read more** button appears at the end of the paragraph.
```

File: docs/write-blog-posts.md [hunk 3]
```diff
@@ -33,6 +33,8 @@ Replace these parameters with real values:
...
 
+>**NOTE:** In `{CONTENT}` must be included `<!-- overview -->` comment which indicates the part that will be shown on the blog's main page. The `Read More` button will appear shortly after that.
```
Comment (klaudiagrz) [inline docs/write-blog-posts.md]:
```suggestion
>**NOTE:** The `<!-- overview -->` comment cannot be placed at the beginning of the blog post. In such a case, the **Read more** button appears straight after the title of the blog post and the content itself is not displayed.
```

**Pull request merged**